f/fluent-bit/build_info.json

```diff
@@ -0,0 +1,16 @@
+{
+    "package_name" : "fluent-bit",
+    "github_url": "https://github.com/fluent/fluent-bit",
+    "version": "2.0.11",
+    "package_dir": "f/fluent-bit",
+    "default_branch": "master",
+    "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
+    "build_script": "fluent-bit-v2.0.11-ubi8.6.sh",
+    "docker_build": "true",
+    "validate_build_script": "true",
+
+    "2.0.11": {
+        "dir": "2.0.11_ubi_8",
+        "build_script": "fluent-bit-v2.0.11-ubi8.6.sh"
```

Comment:
Can we make build script file name using separator `_` instead of  `-`. So it will be like fluent-bit_v2.0.11_ubi8.6.sh. Same for patch file. As I shared earlier `elasticsearch` is best example to refer for this - https://github.com/ppc64le/build-scripts/tree/master/e/elasticsearch

Also we have enabled ubi 8.7 container environment in travis check as well as currency jenkins, so if you can validate your script for 8.7 and include that in script file name.

Reply:
Done.